src/content/news/2025-06-02-maplibre-newsletter-june-2025/index.mdx

```diff
@@ -0,0 +1,244 @@
+date: "2025-06-02"
+categories: ["newsletter"]
+authors: [bart, harel, mpericay, frank, ramyaragupathy]
+draft: false
+---
+
+import { Image } from "astro:assets";
+import mieruneLogo from "/public/img/mierune-logo.svg";
+import maplibreLogo from "/public/img/maplibre-logos/maplibre-logo-for-light-bg.svg";
+
+Welcome to the May edition of the MapLibre newsletter!
+
+This month's updates range from seamless Cloud Optimized GeoTIFF support in GL JS to fresh releases for Martin Tile Server and native SDKs, and the addition of a new team member.
+
+A special thanks to [Mierune](https://www.mierune.co.jp/) for renewing their sponsorship for 2025! Now in its fourth consecutive year, their support has played a vital role in sustaining MapLibre's growth and development.
+
+<div
+  style={{
+    backgroundColor: "white",
+    padding: "2rem",
+    border: "10px solid #cfd8dc",
+    borderRadius: "0",
+    marginBottom: "3rem",
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    gap: "2rem",
+    flexWrap: "nowrap",
+    overflow: "hidden",
+    flexDirection: "column",
+  }}
+>
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "center",
+      alignItems: "center",
+      gap: "2rem",
+      flexWrap: "nowrap",
+      width: "100%",
+    }}
+  >
+    <Image
+      src={maplibreLogo}
+      alt="MapLibre logo"
+      style={{
+        width: "40%",
+        maxWidth: "300px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+    <Image
+      src={mieruneLogo}
+      alt="Mierune logo"
+      style={{
+        width: "40%",
+        maxWidth: "330px",
+        height: "auto",
+        objectFit: "contain",
+      }}
+    />
+  </div>
+  <div
+    style={{
+      marginTop: "1rem",
+      fontSize: "1rem",
+      textAlign: "center",
+      color: "#263238",
+      fontStyle: "italic",
+    }}
+  >
+    Celebrating four amazing years of support from Mierune (2022–2025) 🙌
+  </div>
+</div>
+
+We’ve also published our <a href="about/reports/Financial-Report-Q1-2025.pdf" target="_blank">Q1 2025 Financial Report</a>, continuing our commitment to transparency and open governance.
+Read on to catch up on updates from last month and a glimpse of what's ahead.
+
+## 📱 MapLibre Native
+
+**Plugin Layers are coming:** A major step towards customizability, plugin layers will soon allow developers to register custom layers with their own style spec extensions. The [design proposal](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2025-05-08-plugin-layers.md) was merged this month, and the [implementation PR](https://github.com/maplibre/maplibre-native/pull/3430) led by _**Malcolm Toon of HudHud**_ is nearing completion. HudHud also released their [navigation app](https://apps.apple.com/us/app/hudhud-maps/id6477774492) this month, which showcases MapLibre Native in production.
+
+Note: The first version of plugin layer is _**iOS only**_ and comes with a few limitations.
```

Comment:
```suggestion
Note: The first version of plugin layer is _**iOS only**_ and comes with a few limitations.
```

There is not one plugin layer, so this is incorrect. Could you stick to my wording?

Reply:
Sure, will retain the same.